FEXCore/Source/Interface/Core/ArchHelpers/Arm64Emitter.cpp

```diff
@@ -144,6 +236,101 @@ namespace x32 {
     FEXCore::ARMEmitter::VReg::v28, FEXCore::ARMEmitter::VReg::v29,
     FEXCore::ARMEmitter::VReg::v30, FEXCore::ARMEmitter::VReg::v31
   };
+
```

Comment:
Can we deduplicate this chunk of code? failing to deduplicate stuff in FEX has bit me several times. I don't see why this instance should be different

Reply:
I have a comment there lamenting that I wish we could constexpr to deduplicate but we lack the library/compiler support for now. We could potentially switch over to parsing a uint32_t mask but that would require refactoring/duplicating a bunch of the push/pop routines to support one or the other.